app/src/test/java/org/mozilla/fenix/gleanplumb/state/MessagingMiddlewareTest.kt

```diff
@@ -245,42 +245,63 @@ class MessagingMiddlewareTest {
     }
 
     @Test
-    fun `WHEN updateMessage THEN update available messages`() = runTestOnMain {
+    fun `WHEN updateMessage THEN update available messages in same order`() = runTestOnMain {
         val oldMessage = Message(
             "oldMessage",
-            MessageData(),
+            MessageData(surface = MessageSurfaceId.HOMESCREEN),
```

Comment:
This SurfaceId depends on the generated class based on surface IDs that can change. Maybe we shouldn't use this value?

Same for the `surface` attribute.